src/types/venues.ts

```diff
@@ -141,6 +141,7 @@ export interface BaseVenue {
   code_of_conduct_questions: Question[];
   owners: string[];
   iframeUrl?: string;
+  autoPlay?: boolean;
```

Comment:
Should this be defined in the two places `EmbeddableVenue` and `BaseVenue`?

It can either be property for all venues or just for embeddable venues. Redundant to have it in both.

Similar question with `JazzbarVenue`.

Reply:
fixed in 418cdcb4763615a45870bc6b1dfd26c2ec198a84